config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'posts#index'
+
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] For the post(s) to be able to belong to a user, it must be nested inside a user resource. That's how you can maintaine RESTful url. Kindly check this exmple.

````
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
````